doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -2619,6 +2619,11 @@ Tactic notations allow customizing the syntax of tactics.
         - a term
         - :tacn:`exact`
 
+      * - ``open_constr``
+        - :token:`one_term`
+        - a term where all `_` become evars
```

Comment:
That's not really correct, the holes can be solved by unification. The difference between constr and open_constr is that open_constr allows new holes to exist (ie difference between apply and eapply).

BTW `exact` doesn't actually use `constr`, because using `constr` would mean it gets typechecked without the expected type.

Reply:
Can we also say whether or not typeclass resolution happens with `open_constr`?  (I seem to recall there was at least one version of Coq where there were versions of `open_constr` both with and without TC, and I can't recall which one got picked)
```suggestion
        - a term where all `_` which are not resolved by unification (including canonical structures and typeclass resolution) become evars
```
or
```suggestion
        - a term where all `_` which are not resolved by unification become evars; typeclass resolution is not triggered
```
(maybe with hyperlinks to the corresponding sections?)

Reply:
> BTW `exact` doesn't actually use `constr`, because using `constr` would mean it gets typechecked without the expected type.

Probably a better example is `pose`, though I guess internally that uses `open_constr` and then enforces lack-of-holes at the end.  

Or we could say `pattern`
https://github.com/coq/coq/blob/e7e74b4c490b2e95ed0fd0fa3bb51334aeda6665/user-contrib/Ltac2/Notations.v#L388-L389 or `unify`
https://github.com/coq/coq/blob/e7e74b4c490b2e95ed0fd0fa3bb51334aeda6665/user-contrib/Ltac2/Notations.v#L608